### PR TITLE
`Development`: Link testserver badges in the Pull Request template

### PR DESCRIPTION
**Pull request by BaumiCoder**
#### General

- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/development-process.html#naming-conventions-for-github-pull-requests).

#### Server
only changes in the Pull Request template

#### Client
only changes in the Pull Request template

### Motivation and Context

Imagine you want to test a Pull Request (PR) on a test server:
1. You check the badges at *Testserver States* in your PR.
2. You add the `deploy` label for an available testserver.
3. You want to open the testserver in your webbrowser to do your testing.
4. The badges contain the address, but as an Image. :arrow_right: Badges / Buttons are clickable.
5. A click on the badges only open it as an image in a new browser tab 🤨

### Description

Change the PR template. Now the badges link to there corresponding testservers.

The link is opened in the same browser tab by default. I could not find a way to open a new tab for GitHub Markdown.
The classic HTML `target="_blank"` does not work here.
e.g.  `<a href="https://example.com" target="_blank">example.com</a>` :arrow_right: <a href="https://example.com" target="_blank">example.com</a>

[](https://artemis-test1.artemis.cit.tum.de)
[](https://artemis-test2.artemis.cit.tum.de)
[](https://artemis-test3.artemis.cit.tum.de)
[](https://artemis-test4.artemis.cit.tum.de)
[](https://artemis-test5.artemis.cit.tum.de)
[](https://artemis-test6.artemis.cit.tum.de)
[](https://artemis-test9.artemis.cit.tum.de)

#### Code Review
- [x] Code Review 1
- [x] Code Review 2

## Summary by CodeRabbit

- **Chores**
  - Updated badge URLs in the pull request template to link directly to specific test servers for easier access.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (BaumiCoder) [reply .github/PULL_REQUEST_TEMPLATE.md:84]: An alt text is not helpful here. The important part of the image is the status, which is dynamically. I think it is not possible to show the status in the alt text.

Comment (BaumiCoder) [reply .github/PULL_REQUEST_TEMPLATE.md:84]: Out of scope for this PR.

**Comment by pzdr7** [comment]
I like this change 👍 Please note that we're currently activating TS9 (#8755) - maybe we should wait until that PR is merged before finalizing this one.

**Review by sarpsahinalp — COMMENTED** [review]
Lgtm but as Patrik mentioned waiting for the PR with TS9 to merge is a good idea

**Review by marlon-luca-bu — APPROVED** [review]
Really nice idea 👍

**Review by DominikRemo — APPROVED** [review]
LGTM 🚀

**Review by laadvo — APPROVED** [review]
The links to the test servers are working as expected

**Review by coolchock — APPROVED** [review]
lgtm

**Review by sarpsahinalp — DISMISSED** [review]

**Comment by BaumiCoder** [comment]
> I like this change 👍 Please note that we're currently activating TS9 (#8755) - maybe we should wait until that PR is merged before finalizing this one.

@pzdr7, @sarpsahinalp Okay will wait for #8755 to be merged, and then I will add a link to test server 9.

> We also have ts 7,8 and 9(as Patrik mentioned). Are test servers 7 and 8 reserved?

I don't know why there is a gap, but at the moment they do not exist (Host not found for [artemis-test7](https://artemis-test7.artemis.cit.tum.de/) and [artemis-test8](https://artemis-test8.artemis.cit.tum.de/)). Also, the Badges are not defined (see https://byob.yarr.is/ls1intum/Artemis/artemis-test7 and https://byob.yarr.is/ls1intum/Artemis/artemis-test8)

**Review by SimonEntholzer — APPROVED** [review]

**Review by pzdr7 — APPROVED** [review]
Thanks for waiting. LGTM 👍

**Review by JohannesStoehr — APPROVED** [review]
Approve after merge for TS9

**Review by magaupp — APPROVED** [review]
I like this change. I always confused myself when I opened the badge image :sweat_smile: 
Code LGTM

**Review by egekurt123 — APPROVED** [review]
LGTM

**Review by DominikRemo — APPROVED** [review]
Reapprove

**Review by laadvo — APPROVED** [review]
Reapprove

**Review by coolchock — APPROVED** [review]
re-approve

**Pull request merged**
